.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,9 +1,9 @@
-**This project uses [semver](http://semver.org), please check the scope of this pr:**
+### This project uses [semver](semver.org), please check the scope of this pr:
```

Comment:
I'm good with this. As long as we are aware that we need to fix that later. (it's a very minor thing)